materialize-boilerplate/boilerplate.go

```diff
@@ -59,6 +55,31 @@ type cmdCommon struct {
 	srv pm.DriverServer
 	r   *bufio.Reader
 	w   protoio.Writer
+
+	logging *logConfig
+}
+
+// logConfig configures handling of application log events.
+type logConfig struct {
+	Level  string `long:"log.level" env:"LOG_LEVEL" default:"info" choice:"info" choice:"INFO" choice:"debug" choice:"DEBUG" choice:"warn" choice:"WARN" description:"Logging level"`
+	Format string `long:"log.format" env:"LOG_FORMAT" default:"text" choice:"json" choice:"text" choice:"color" description:"Logging output format"`
+}
+
+func (c *cmdCommon) Execute(args []string) error {
```

Comment:
nit: Common.Execute() just sets logger params. Can we call it ParseArgs() or something more explicit?

Reply:
Yeah, that naming was a holdover from when I was also moving the TCP socket opening into that function. I've renamed the method and changed the receiver type so the usage looks like `c.cmdCommon.logging.Configure()` now.